### PR TITLE
Added `function`

### PR DESCRIPTION
**Pull request by ufik**

### CODE REVIEW

File: components/Translation/basic_usage.rst [hunk 1]
```diff
@@ -21,12 +21,12 @@ First let's create a class which will keep our translatable properties:
        /**
         * @var string
         */
-       private $title;
+       private function $title;
```
Comment (michalmarcinkowski) [inline components/Translation/basic_usage.rst]: This should be reverted

Comment (NeverResponse) [inline components/Translation/basic_usage.rst]: Overall good catch with missing `function`'s, but you took a little too far, this isn't a function, but a variable.
Other than that :+1:

**Comment by ufik** [comment]
My bad, now fixed.

**Comment by michalmarcinkowski** [comment]
Good catch, thanks Tomáš!

**Pull request merged**